ynh_install_nodejs/ynh_install_nodejs

```diff
@@ -0,0 +1,301 @@
+#
+#
+# one other variable is also available
+#   - $nodejs_path: The absolute path to Node.js binaries for the chosen version.
+#
+# usage: ynh_use_nodejs
+#
+# Requires YunoHost version 2.7.12 or higher.
+ynh_use_nodejs () {
+    nodejs_version=$(ynh_app_setting_get --app=$app --key=nodejs_version)
+
+    # Get the absolute path of this version of Node.js
+    nodejs_path="$nodejs_version_path/$YNH_APP_INSTANCE_NAME/bin"
+
+    # Allow alias to be used into bash script
+    shopt -s expand_aliases
+
+    # Create an alias for the specific version of Node.js and a variable as fallback
+    ynh_node="$nodejs_path/node"
+    alias ynh_node="$ynh_node"
+    # And npm
+    ynh_npm="$nodejs_path/npm"
+    alias ynh_npm="$ynh_npm"
+
+    # Load the path of this version of Node.js in $PATH
+    if [[ :$PATH: != *":$nodejs_path"* ]]; then
+        PATH="$nodejs_path:$PATH"
+    fi
+    # Create an alias to easily load the PATH
+    ynh_node_load_PATH="PATH=$PATH"
+    ynh_nodejs_load_path="PATH=$PATH"
+
+    # Sets the local application-specific Node.js version
+    pushd $final_path
+        $nodenv_install_dir/bin/nodenv local $nodejs_version
+    popd
+}
+
+# Install a specific version of Node.js
+#
+# ynh_install_nodejs will install the version of Node.js provided as argument by using nodenv.
+#
+# This helper creates a /etc/profile.d/nodenv.sh that configures PATH environment for nodenv
+# for every LOGIN user, hence your user must have a defined shell (as opposed to /usr/sbin/nologin)
+#
+# Don't forget to execute Node.js-dependent command in a login environment
+# (e.g. sudo --login option)
+# When not possible (e.g. in systemd service definition), please use direct path
+# to nodenv shims (e.g. $NODENV_ROOT/shims/bundle)
+#
+# usage: ynh_install_nodejs --nodejs_version=nodejs_version
+# | arg: -v, --nodejs_version= - Version of Node.js to install.
+#
+# Requires YunoHost version 2.7.12 or higher.
+ynh_install_nodejs () {
+    # Declare an array to define the options of this helper.
+    local legacy_args=v
+    local -A args_array=( [v]=nodejs_version= )
+    local nodejs_version
+    # Manage arguments with getopts
+    ynh_handle_getopts_args "$@"
+
+    # Load nodenv path in PATH
+    local CLEAR_PATH="$nodenv_install_dir/bin:$PATH"
+
+    # Remove /usr/local/bin in PATH in case of Node.js prior installation
+    PATH=$(echo $CLEAR_PATH | sed 's@/usr/local/bin:@@')
+
+    # Move an existing Node.js binary, to avoid to block nodenv
+    test -x /usr/bin/node && mv /usr/bin/node /usr/bin/node_n
+    test -x /usr/bin/npm && mv /usr/bin/npm /usr/bin/npm_n
+
+    # Install or update nodenv
+    nodenv="$(command -v nodenv $nodenv_install_dir/bin/nodenv | head -1)"
+    if [ -n "$nodenv" ]; then
+        ynh_print_info --message="nodenv already seems installed in \`$nodenv'."
+        pushd "${nodenv%/*/*}"
+            if git remote -v 2>/dev/null | grep "https://github.com/nodenv/nodenv.git"; then
```

Comment:
How do you handle the case where this git remote doesn't exist?

Reply:
it's not managed :/

But wasn't managed on original helper: 
https://github.com/YunoHost/yunohost/blob/adc83b4c9c2c30e9ef75f3609c538b646f91f1db/data/helpers.d/nodejs#L16-L27

You think it's mandatory to manage it ?